pyccl/pkresponse.py

```diff
@@ -42,20 +42,20 @@ def Pmm_resp(
         extra_parameters (:obj:`dict`): Dictionary holding extra
             parameters. Currently supports extra parameters for CAMB.
         a_arr (array): an array holding values of the scale factor
-            at which the trispectrum should be calculated for
-            interpolation.
+            at which the response is calculated.
         lk_arr (array): an array holding values of the natural
             logarithm of the wavenumber (in units of Mpc^-1) at
-            which the trispectrum should be calculated for
-            interpolation.
+            which the response is calculated.
 
     Returns:
         Response of the matter power spectrum.
     """
 
-    # Make sure input makes sense
-    if (a_arr is None) or (lk_arr is None):
-        raise ValueError("you must provide arrays")
+    # Set k and a sampling from CCL parameters
+    if a_arr is None:
+        a_arr = get_pk_spline_a()
```

Comment:
Thanks! I would use the same code that was there before, just in case there is a subtle difference that we are missing
```
    if lk_arr is None:
        status = 0
        nk = lib.get_pk_spline_nk(cosmo.cosmo)
        lk_arr, status = lib.get_pk_spline_lk(cosmo.cosmo, nk, status)
        check(status, cosmo=cosmo)
    if a_arr is None:
        status = 0
        na = lib.get_pk_spline_na(cosmo.cosmo)
        a_arr, status = lib.get_pk_spline_a(cosmo.cosmo, na, status)
        check(status, cosmo=cosmo)
```